src/ios/hooks/ab/add-targets.js

```diff
@@ -166,55 +175,94 @@ module.exports = function (context) {
 
     var watchKitExtensionResourcesBuildPhase = pbxProject.addBuildPhase([watchKitAppFile.path], 'PBXResourcesBuildPhase', watchKitExtension + ' Resources');
```

Comment:
Is this phase needed? We don't add it to the new target anymore: https://github.com/Telerik-Verified-Plugins/AppleWatch/pull/19/files#diff-fe59b2a77283faf286762cee9f417ba3R202